xslt/geonetwork_update_backlogs_324/transform-mcp-1.4.xsl

```diff
@@ -0,0 +1,191 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="2.0"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:mcp="http://bluenet3.antcrc.utas.edu.au/mcp"
+                xmlns:gco="http://www.isotc211.org/2005/gco"
+                xmlns:gmd="http://www.isotc211.org/2005/gmd"
+                xmlns:gmx="http://www.isotc211.org/2005/gmx"
+                xmlns:geonet="http://www.fao.org/geonetwork"
+
+                exclude-result-prefixes="xsl mcp gco gmd gmx geonet"
+>
+
+
+    <xsl:template match="node()|@*">
+        <xsl:copy>
+            <xsl:apply-templates select="node()|@*"/>
+        </xsl:copy>
+    </xsl:template>
+
+    <xsl:template match="gmd:URL/text()[.='http://i.creativecommons.org/l/by/4.0/88x31.png']">https://licensebuttons.net/l/by/4.0/88x31.png</xsl:template>
+
+    <xsl:template match="mcp:MD_Metadata/gmd:identificationInfo/mcp:MD_DataIdentification/gmd:pointOfContact">
+        <xsl:choose>
+            <xsl:when test="gmd:CI_ResponsibleParty/gmd:organisationName/gco:CharacterString/text()[.='Integrated Marine Observing System (IMOS)']">
+                <xsl:copy-of select="$aodnPointOfContact"/>
+                <!--<xsl:comment>Match!</xsl:comment>-->
+            </xsl:when>
+            <xsl:otherwise>
+                <xsl:copy-of select="."/>
+                <!--<xsl:comment>No Match!</xsl:comment>-->
+            </xsl:otherwise>
+        </xsl:choose>
+    </xsl:template>
+
+    <xsl:template match="mcp:MD_Metadata/gmd:contact">
+        <xsl:choose>
+            <xsl:when test="gmd:CI_ResponsibleParty/gmd:organisationName/gco:CharacterString[matches( text(), 'Integrated Marine Observing System (IMOS)|eMarine Information Infrastructure (eMII)|eMII project|Derwent Estuary Program' )]">
+                <xsl:copy-of select="$aodnContact"/>
+                <!--<xsl:comment>Match!</xsl:comment>-->
+            </xsl:when>
+            <xsl:otherwise>
+                <xsl:copy-of select="."/>
+                <!--<xsl:comment>No Match!</xsl:comment>-->
+            </xsl:otherwise>
+        </xsl:choose>
+    </xsl:template>
+
+    <xsl:variable name="aodnPointOfContact">
```

Comment:
Would be better if this was included rather than duplicated in the two xsl's - have to check this text twice.